src/AzureAccountWrapper.ts

```diff
@@ -79,8 +79,18 @@ export class AzureAccountWrapper {
     }
 
     async query(path: string, apiVersion: string = DEFAULT_API_VERSION): Promise<any> {
+        interface PathVersion {
+            path: string;
+            version: string;
+        }
+        const customApiVersions: PathVersion[] = vscode.workspace
+            .getConfiguration("azure-resource-explorer-for-vscode")
+            .get("customApiVersions") ?? [];
+
+        const match = customApiVersions.find((x :PathVersion) => x.path === path);
```

Comment:
Consider doing a regex match instead - would be easier to configure.

Reply:
I can't think of a way to use regex here, could you be more specific?

Reply:
```
const match = customApiVersions.find((x :PathVersion) => new RegExp(x.path).test(path));
```

That way it will be much easier to configure customApiVersions - no need to specify full paths (with subscriptionIds etc.)

Reply:
I went with `pathIncludes`, I tested it with a few different possibilities, it works! I also updated the README